src/components/menu/styles.ts

```diff
@@ -18,3 +18,55 @@ export const ElMenuList = styled.div`
   background-color: var(--fill-white);
   box-shadow: 0px 4px 16px 0px #222b3329;
 `
```

Comment:
**issue:** I don't see any handling of the following:

1. Disabled menu items (which according to https://www.w3.org/WAI/ARIA/apg/patterns/menubar/ is handled via `aria-disabled`);
2. Focus state for menu items.

Reply:
> issue: I don't see any handling of the following:
Disabled menu items (which according to https://www.w3.org/WAI/ARIA/apg/patterns/menubar/ is handled via aria-disabled)

yes, I also don't see in the design spec have that at the moment, 

> Focus state for menu items

I think its because the planning it to be put it in the app bar, which close the menu on click,  I haven't tought about that, I will add it here using `:hover` style 👍

Reply:
> yes, I also don't see in the design spec have that at the moment,

Yes, the design spec is not complete. That doesn't mean we don't need to facilitate standard concerns like disabled states and focus states.

> I think its because the planning it to be put it in the app bar, which close the menu on click, I haven't tought about that, I will add it here using :hover style 👍

Sorry, but I don't understand what the app bar has to do with `:focus` or `:focus-visible` styling of menu items in a menu. Can you clarify please?

Reply:
> Yes, the design spec is not complete. That doesn't mean we don't need to facilitate standard concerns like disabled states and focus states.

👍 

> Sorry, but I don't understand what the app bar has to do with :focus or :focus-visible styling of menu items in a menu. Can you clarify please?

when user click the menu button in the appbar to trigger menu open, then a11y suggest to add a handler when user click the `arrowDown` key, it will focus to the first element in the menu, in that case `:focus` style will be applied, which in our case I will use `:hover` or `Selected` state in design, can I?

Reply:
ok, I have confirmed with design team, and they agree https://www.figma.com/design/6CaivqdlTX0UkFYJkpBKDu?node-id=83-2711&m=dev#1002341758